tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.AppSec.cs

```diff
@@ -74,6 +74,17 @@ internal class AppSec
             /// Automatic tracking of user events mode. Values can be disabled, safe or extended.
             /// </summary>
             internal const string UserEventsAutomatedTracking = "DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING";
+
+            /// <summary>
+            /// Percentage of requests for which the schema should be extracted. Between 0 and 1, defaults to 0.1 (10%)
+            /// A value of 0 means no schemas are extracted, effectively disabling schema extraction altogether
+            /// </summary>
+            internal const string ApiSecurityRequestSampleRate = "DD_API_SECURITY_REQUEST_SAMPLE_RATE";
+
+            /// <summary>
+            /// Unless set to true or 1, tracers don’t collect schemas. After the experiment, the environment variable will be removed and schema collection will be enabled only when ASM is enabled
+            /// </summary>
+            internal const string ApiExperimentalSecurityEnabled = "DD_EXPERIMENTAL_API_SECURITY_ENABLED";
```

Comment:
Are these keys standardised across languages?

Reply:
the experimental flag should go, and the sample rate is, but yes all languages will implement / have implemented this way